### PR TITLE
Update memory settings for Docker for Mac

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

File: presto-product-tests/README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+To increase docker memory go to docker preferences > advanced > memory, and increase it appropriately.
```
Comment (amrutagokhale) [inline presto-product-tests/README.md]: Suggestion:
"To increase docker memory, go to Docker Preferences -> Advanced -> Memory."

File: presto-product-tests/README.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+To increase docker memory, go to Docker Preferences > Advanced > Memory, and increase it appropriately.
+We recommend at least 6GB for such profiles.
```
Comment (amrutagokhale) [inline presto-product-tests/README.md]: Why don't you rearrange it like this:
``The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap. We recommend at least 6GB for such profiles. You can increase docker memory by following Docker Preferences -> Advanced -> Memory.
 ``

**Review by amrutagokhale — CHANGES_REQUESTED** [review]

File: presto-product-tests/README.md (hunk 2 above)
Comment (cawallin) [reply presto-product-tests/README.md]: What Amruta said, but  "by going to Docker Preferences"

File: presto-product-tests/README.md [hunk 3]
```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+To increase docker memory go to docker preferences > advanced > memory, and increase it appropriately.
+We recommend at least 6GB for such profiles.
```
Comment (cawallin) [inline presto-product-tests/README.md]: Either change to 4GB or remove

Comment (sanjay990) [reply presto-product-tests/README.md]: I'll remove this.

**Review by amrutagokhale — APPROVED** [review]

**Review by cawallin — APPROVED** [review]

**Pull request closed without merging**